codespell_lib/data/dictionary.txt

```diff
@@ -10074,6 +10074,9 @@ dnamically->dynamically
 dne->done
 do'nt->don't
 doalog->dialog
+doamin->domain, dopamine,
+doamine->dopamine
```

Comment:
It's probably worth suggesting domain here too as a secondary one I'd have thought.

Reply:
I agree